frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/grid_page.dart

```diff
@@ -221,28 +213,30 @@ class _GridRowsState extends State<_GridRows> {
               },
             );
           },
-          buildWhen: (previous, current) {
-            return current.reason.maybeWhen(
-              reorderRows: () => true,
-              reorderSingleRow: (reorderRow, rowInfo) => true,
-              delete: (item) => true,
-              insert: (item) => true,
-              orElse: () => false,
-            );
-          },
+          buildWhen: (previous, current) => current.reason.maybeWhen(
+            reorderRows: () => true,
+            reorderSingleRow: (reorderRow, rowInfo) => true,
+            delete: (item) => true,
+            insert: (item) => true,
+            orElse: () => false,
+          ),
           builder: (context, state) {
             final rowInfos = context.watch<GridBloc>().state.rowInfos;
 
-            return SliverFillRemaining(
+            return ScrollConfiguration(
+              behavior:
+                  ScrollConfiguration.of(context).copyWith(scrollbars: false),
               child: ReorderableListView.builder(
+                /// TODO(Xazin): Resolve inconsistent scrollbar behavior
+                ///  This is a workaround related to
+                ///  https://github.com/flutter/flutter/issues/25652
+                cacheExtent: 5000,
                 key: _key,
```

Comment:
This key is used for animation and it should be associated with the `SliverAnimatedListState`.

You changed `SliverFillRemaining` to `ReorderableListView`, so the animation will no longer take effect.

Reply:
I suppose the logic of animation can be removed now.